src/manifolds/GrassmannProjector.jl

```diff
@@ -0,0 +1,91 @@
+@doc raw"""
+    ProjectorPoint <: AbstractManifoldPoint
+
+A type to represent points on a manifold that are orthogonal projectors.
```

Comment:
```suggestion
A type to represent points on a manifold [`Grassmann`](@ref) that are orthogonal projectors.
```
Is this only for the Grassmann manifold?